tools/wptserve/tests/functional/docroot/test_h2_headers_data.py

```diff
@@ -3,4 +3,4 @@ def handle_headers(frame, request, response):
     response.headers.update([('test', 'passed')])
 
 def handle_data(frame, request, response):
-    response.content = frame.data[::-1]
+    response.content.append(frame.data.swapcase())
```

Comment:
Yes, it's because httpx sends two data frames, first the 'hello world!' and then an empty one. I guess this is because of some accident in the implementation and it could change. But still, it's better to not depend on how the data is sliced, and the new code would work for any slicing.